data/io.elementary.terminal.gschema.xml

```diff
@@ -91,7 +91,7 @@
     </key>
 
     <key name="foreground" type="s">
-      <default>"#94a3a5"</default>
```

Comment:
This color is still being referenced in `io.elementary.terminal.gschema.xml`, as part of "palette", and in the method `TerminalWidget.restore_settings`. Should these be replaced as well, or should the palette be left untouched by this PR? (Not sure why there is both a palette, and default fg/bg colors.)

Reply:
@jlnr hm, I'm not sure. I believe the palette is used by the VTE itself for things like curses. I'm not sure if it should be updated.

Reply:
Fair enough. If it looks odd in real-world use cases, we can open another PR to fix things.